compiler/rustc_expand/src/expand.rs

```diff
@@ -1715,9 +1740,22 @@ impl<'a, 'b> InvocationCollector<'a, 'b> {
             return match self.take_first_attr(&mut node) {
                 Some((attr, pos, derives)) => match attr.name_or_empty() {
                     sym::cfg => {
-                        if self.expand_cfg_true(&mut node, attr, pos) {
+                        let (res, meta_item) = self.expand_cfg_true(&mut node, attr, pos);
+                        if res {
                             continue;
                         }
+
+                        if let Some(meta_item) = meta_item {
+                            let parent_module = self.cx.current_expansion.lint_node_id;
+
+                            for name in node.declared_names() {
+                                self.cx.resolver.append_stripped_out_item(
+                                    parent_module,
```

Comment:
```suggestion
                                    self.cx.current_expansion.lint_node_id,
```
(Not necessarily a module too.)